File: ShipperManagerAndroid/app/src/main/java/com/example/shippermanager/Model/ChiTietDonHang.java
package com.example.shippermanager.Model;

import java.io.Serializable;

public class ChiTietDonHang implements Serializable {

    public SanPham SanPham;
    public int SoLuong;

    public ChiTietDonHang(SanPham sanPham) {
        this.SanPham = sanPham;
        this.SoLuong = 1;
    }

    public ChiTietDonHang() {

    }

    public double GetMoneySum()
    {
        return SanPham.Gia * SoLuong;
    }

}

File: ShipperManagerWeb/ShipperManagerWeb/ShipperManager/Models/ChiTietDonHang.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.Web;

namespace ShipperManager.Models
{
    public class ChiTietDonHang
    {

        public int SoLuong { get; set; }
        public SanPham SanPham { get; set; }

        public ChiTietDonHang(SanPham sp)
        {

            SoLuong = 1;
            SanPham = sp;
        }

        public ChiTietDonHang()
        {
        }

        public decimal GetTongTien()
        {
            return SanPham.Gia * SoLuong;
        }

    }
}